.yamllint.yml

```diff
@@ -0,0 +1,18 @@
+extends: default
+
+ignore-from-file: .gitignore
+
+rules:
+  document-start:
+    present: false
+
+  # 80 chars should be enough, but don't fail if a line is longer
+  line-length:
+    max: 80
+    level: warning
+    allow-non-breakable-words: true
+    allow-non-breakable-inline-mappings: true
+
+  # don't bother me with this rule
+  indentation: disable
+  comments: disable
```

Comment:
What do you think about:

```
rules:
  comments:
    require-starting-space: true
```

? Open to discussion.

Reply:
Ok not bad, we can do it